deno.jsonc

```diff
@@ -1,8 +1,28 @@
 {
+  "name": "@shougo/ddc-vim",
+  "version": "0.0.0",
+  "exports": {
+    ".": "./denops/ddc/deps.ts",
```

Comment:
`deps.ts` should not be exported. (I think if the module exports `.`, `types.ts` is better.)

```suggestion

```

Reply:
Removed now